client/starwhale/api/_impl/data_store.py

```diff
@@ -417,6 +417,8 @@ def __init__(self, raw_type: Type, attrs: Dict[str, SwType]) -> None:
         self.attrs = attrs
 
     def merge(self, type: SwType) -> SwType:
+        if type is UNKNOWN or self is type:
```

Comment:
add some specified unit tests for this change in the test_datastore.py?

Reply:
> add some specified unit tests for this change in the test_datastore.py?

added